Cluster_Display_Use_Case_v2/stpa/purpose.md

```diff
@@ -0,0 +1,83 @@
+# Define the purpose of analysis
+
+## Goals
+
+> What kinds of losses will the analysis aim to prevent? Will STPA be applied
+only to traditional safety goals like preventing loss of human life or will it
+be applied more broadly to security, privacy, performance, and other system
+properties? What is the system to be analyzed and what is the system boundary?
+
+## System boundary
+
+### Definitions
+> A system is a set of components that act together as a whole to achieve some
+  common goal, objective, or end. A system may contain subsystems and may also be
+  part of a larger system
+> A system is an abstraction conceived by the analyst. A decision must be made
+about what is included in the system and what the system boundary is. With
+respect to engineering, the most useful way to define the system boundary for
+analysis purposes is to include the parts of the system over which the system
+designers have some control.
+
+![Control Structure Diagram](system-context.dot.png)
+
+## Notes
+
+* For next level down, include another application (e,g, IVI) running on the
+  same system as the Instrument Cluster
+    - This may have a second display and other peripheral components
+
+### Questions
+
+* What are the telltale requesters?
+    - These would help us to characterise the possible vehicle-level hazards
+* Is there an IVI running on the same SoC as the Cluster?
+    - Is a Hypervisor involved in the system?
+* Are there any other external controllers?
+
+## Losses
+
+### Definitions
+> A loss involves something of value to stakeholders. Losses may include a loss
+  of human life or human injury, property damage, environmental pollution, loss
+  of mission, loss of reputation, loss or leak of sensitive information, or any
+  other loss that is unacceptable to the stakeholders.
+
+* L-1: Loss of life or injury to people
+* L-2: Loss of or damage to vehicle  
+* L-3: Loss of or damage to objects outside the vehicle
+
```

Comment:
Not sure if we finished discussing this during the call, but do we want to add Loss of function here as well as a non-safety relevant loss?

Reply:
Chris recommended that we didn't include this, as we want to keep a clear focus on safety